src/main/resources/messages_es.properties

```diff
@@ -252,17 +252,17 @@ language-preference.language-to-read-label=
 language-preference.language-to-read-help=El DCFS har� todo lo que est� a su alcance para brindarle los documentos en su idioma de preferencia.
 language-preference.need-interpreter-label=�Necesita un int�rprete?
 language-preference.need-interpreter-help=El DCFS har� todo lo que est� a su alcance para ponerlo en contacto con alguien que hable su idioma, incluida la lengua de se�as estadounidense.
-language-preference.english=English
+language-preference.english=Ingl�s
```

Comment:
I dont think these should be translated since it's asking people's language preferences